### PR TITLE
Pal 1076 helm charts include released images

### PR DESCRIPTION
**Pull request by m78233**

### CODE REVIEW

File: user-service/pom.xml [hunk 1]
```diff
@@ -278,6 +278,24 @@
...
+                            <goal>read-project-properties</goal>
+                        </goals>
+                        <configuration>
+                            <files>
+                                <file>${project.basedir}/../charts/${project.artifactId}/values.yaml</file>
+                            </files>
+                        </configuration>
+                    </execution>
+                </executions>
+            </plugin>
```
Comment (pd104923) [inline user-service/pom.xml]: Is there a reason this has been duplicated? It's the same as the plugin above.

**Review by pd104923 — CHANGES_REQUESTED** [review]
Just the 1 question

File: user-service/pom.xml (hunk 1 above)
Comment (m78233) [reply user-service/pom.xml]: No, should not be there.  Have removed it.

**Review by pd104923 — DISMISSED** [review]

File: pom.xml [hunk 2]
```diff
@@ -43,6 +43,7 @@
     <properties>
         <!-- Project properties -->
         <java.version>11</java.version>
+        <version.number>0.5.0</version.number>
```
Comment (dev930018) [inline pom.xml:46]: I'd also expect the project-level `<version/>` tag to be updated appropriately for this parameter:
From
```
    <version>0.5.0-${revision}</version>
```
To
```
    <version>${version.number}-${revision}</version>
```

**Review by dev930018 — CHANGES_REQUESTED** [review]

File: pom.xml (hunk 2 above)
Comment (pd104923) [reply pom.xml:46]: Is this comment from @dev930018 no longer relevant? Looks like this should be updated.

**Review by pd104923 — COMMENTED** [review]
1 small follow up on a previous comment.

File: pom.xml (hunk 2 above)
Comment (m78233) [reply pom.xml:46]: Would have preferred this approach, but this hits up agains an issue. Both version and revision are reserved elements in Maven and can be used in the way that we have in the first example. The element version.number does not exists at the point the version element is being assigned a value and so any attempt to us it fails.

**Review by pd104923 — APPROVED** [review]

**Review by dev930018 — APPROVED** [review]

**Review by ac74475 — APPROVED** [review]

**Pull request merged**